crates/anstyle-svg/src/lib.rs

```diff
@@ -31,6 +31,16 @@ pub use anstyle_lossy::palette::Palette;
 pub use anstyle_lossy::palette::VGA;
 pub use anstyle_lossy::palette::WIN10_CONSOLE;
 
+use std::fmt::Write as _;
+
+/// Contains the different parts of a HTML rendered page.
+pub struct HtmlParts {
```

Comment:
`Term` is the central type for this crate while this is an optional piece.  I'd prefer this to go after `Term`  to help people find the right jumping off point quickly